.readthedocs.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+sphinx:
+  configuration: "./conf.py"
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.8"
+    nodejs: "14"
+  jobs:
+    pre_build:
+      - pip install -r requirements.txt
+      - npm ci
+      - npm run build-html
+    post_build:
+      - rm -rfv ../../artifacts/${READTHEDOCS_VERSION}/sphinx
+      - mv -v ./output/ ../../artifacts/${READTHEDOCS_VERSION}/sphinx
```

Comment:
Haha! Nice hack to support building with pelican 😛

Reply:
Yea, I figured there would be a blog post about how to support other tools like this in <3 months after we promote this.